scripts/lib/setup-apt-repo

```diff
@@ -58,6 +58,8 @@ deb https://packagecloud.io/zulip/server/debian/ stretch main
 deb https://packages.groonga.org/debian/ stretch main
 deb-src https://packages.groonga.org/debian/ stretch main
 EOF
+elif [ "$release" = "buster" ]; then
+    :
```

Comment:
I think we still want to be adding those two repositories, as long as it doesn't crash to do so... more future-proof.  

If it does, we can ensure there are empty repos for buster.